src/controllers/check.company.controller.ts

```diff
@@ -1,16 +1,16 @@
-import {NextFunction, Request, Response} from "express";
-import {PTFCompanyProfile} from "../model/company.profile";
-import {CHECK_COMPANY} from "../model/template.paths";
-import {getPromiseToFileSessionValue} from "../services/session.service";
-import {COMPANY_PROFILE} from "../session/keys";
+import { NextFunction, Request, Response } from "express";
+import { PTFCompanyProfile } from "../model/company.profile";
+import { Templates } from "../model/template.paths";
```

Comment:
Presume that we could also import Templates.CHECK_COMPANY here? If yes, that would avoid repeating 'Templates.' twice further down and be consistent with the COMPANY_PROFILE import too.

Reply:
Wouldn't it read better if it were Templates.CHECK_COMPANY instead of CHECK_COMPANY?

Reply:
I think in these cases it's fairly intuitive, but I don't mind. However, if we do stick with this (as a new "standard") shouldn't you refactor the COMPANY_PROFILE import?

Reply:
this pr was just to do the templates. Not sure about session keys as it will mean changing all the session modules and the plan is to replace it when we get the common component.

Reply:
Tried using import Templates.COMPANY_NUMBER but you can't import the individual type of the Enum, only the enum itself is exported.